components/mapping.sorted

```diff
@@ -0,0 +1,1886 @@
+(origin:  meta-packages/history
```

Comment:
1) How long is it generated?
2) Obviously, there should be some hard-coded values or exception list.

Reply:
1. A few minutes with the Makefile target but this is only for testing and comparing with the userland-mapping tool by @xen0l. I'll remove it before integration.
2. This mapping would be used to map the packages so probably need to add a list of provided packages for non-ips.mk components and overload the target. Otherwise the graph will be missing chunks. How it should be handled is still a question. Hopefully we'll move forward on this today.